src/app/dspace/components/item/full/full-collections.component.ts

```diff
@@ -1,37 +1,32 @@
 import {Component, Input} from 'angular2/core';
+import {ROUTER_DIRECTIVES} from 'angular2/router';
 import {TranslatePipe} from "ng2-translate/ng2-translate";
 
-
-import {Item} from "../../../models/item.model"
+import {Collection} from "../../../models/collection.model";
 import {ViewElementComponent} from '../view-element.component';
 
 /**
  * Component for the collections of the simple-item-view.
- * When you click on the collection name, it has to redirect to the right collection.
  */
 @Component({
     selector: 'item-full-collections',
-    inputs: ['itemData'],
-    directives: [ViewElementComponent],
+    inputs: ['itemParent'],
+    directives: [ROUTER_DIRECTIVES, ViewElementComponent],
     pipes: [TranslatePipe],
-    template:
-        `
-        <view-element [header]="componentTitle | translate">
-            <ul>
-                <li>
-                   <a [attr.href]="collectionURIPrefix+itemData.id">{{itemData.name}}</a>
-                </li>
-            </ul>
-
-        </view-element>
-        `
+    template: `
+                <view-element [header]="componentTitle | translate">
+                    <ul>
+                        <li>
+                            <a *ngIf="itemParent && itemParent.component && itemParent.id" [routerLink]="[itemParent.component, {id: itemParent.id}]">{{ itemParent.name }}</a>
```

Comment:
That is actually a good point, the UL could be used for more in the future.
So best to put it on the LI tag then? 

Yeah there should be a UL I think - because the idea is that an item can have more than one collection and they should all be listed there in the future, not just the parent collection. At least, that is what I'm thinking, but will check with @artlowel on that.